package.json

```
@@ -41,9 +41,9 @@
   },
   "license": "MIT",
   "dependencies": {
-    "@ministryofjustice/fb-client": "2.1.4",
+    "@ministryofjustice/fb-client": "2.1.5",
     "@ministryofjustice/fb-components": "1.4.4",
```

```suggestion
    "@ministryofjustice/fb-components": "1.4.5",
```
Did you want to bump the components as well while you're here?

---

no because bumping components breaks _all_ the runners :(

going to need to go back and figure out what in the components is breaking

---

oh no :(
fair enough, thought I'd check before approving - looks good otherwise 🚀